extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -47,10 +49,12 @@ if (isTestSetupCompleted()) {
 		test('Schema compare database to database comparison, script generation, and scmp @UNSTABLE@', async function () {
 			await schemaCompareTester.SchemaCompareDatabaseToDatabase();
 		});
-		// TODO: figure out why this is failing with Error: This editor is not connected to a database Parameter name: OwnerUri
-		// test('Schema compare dacpac to database comparison, script generation, and scmp', async function () {
-		// 	await schemaCompareTester.SchemaCompareDacpacToDatabase();
-		// });
+		test('Schema compare dacpac to database comparison, script generation, and scmp @UNSTABLE@', async function () {
+			await schemaCompareTester.SchemaCompareDacpacToDatabase();
+		});
+		test('Schema compare dacpac to dacpac comparison with include exclude @UNSTABLE@', async function () {
```

Comment:
for the new one is there an easy way to check if its unstable in lab  and remove tag right away if its stable...?

Reply:
Probably just let in run in the unstable test runs for a week or so then have PR to remove the tag if it's stable. @alanrenmsft is investigating the error that was causing these tests to be unstable.